src/main/java/no/unit/nva/doi/transformer/MainHandler.java

```diff
@@ -63,7 +65,7 @@ public MainHandler(ObjectMapper objectMapper, DataciteResponseConverter datacite
                        CrossRefConverter crossRefConverter, Environment environment) {
         this.objectMapper = objectMapper;
         this.publicationTransformer = new PublicationTransformer(dataciteConverter, crossRefConverter,
-            createObjectMapper());
+                createObjectMapper());
         this.allowedOrigin = environment.get(ALLOWED_ORIGIN).orElseThrow(
```

Comment:
nva-doi-transformer is not using nva-commons? That's worthy of a new task I think